ci/gpu/build.sh

```diff
@@ -128,7 +129,7 @@ GTEST_OUTPUT="xml:${WORKSPACE}/test-results/libcuml_cpp/" ./test/ml
 logger "Python pytest for cuml..."
 cd $WORKSPACE/python
 
-pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v -s -m "not memleak"
+pytest --cache-clear --junitxml=${WORKSPACE}/junit-cuml.xml -v -s -m "not memleak" --durations=50 --timeout=300
```

Comment:
What does `--durations=50` do?

Reply:
It just prints the 50 slowest tests after running all pytests, will allow us to keep track and authors of PRs to know if they’ve introduced slow tests